src/components/diagrams/singleLineDiagram/single-line-diagram.js

```diff
@@ -326,6 +328,24 @@ const SingleLineDiagram = forwardRef((props, ref) => {
     const [svgFinalWidth, setSvgFinalWidth] = useState();
     const [svgFinalHeight, setSvgFinalHeight] = useState();
 
+    useEffect(() => {
+        if (finalPaperHeight) {
+            setDisplayedSldHeights((displayedSldHeights) => {
+                if (
+                    displayedSldHeights.filter((sld) => sld.id === sldId)
+                        .length > 0
```

Comment:
displayedSldHeights.find(sld => sld.id === sldId) ?

Reply:
I removed this bit